src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -69,15 +51,15 @@ const ProgressBar = ({
                         <b>{numGiftCards}</b> vouchers
                       </td>
                       <td>
-                        <b>${Math.floor(giftCardAmount) / 100}</b>
+                        <b>${Math.floor(giftCardAmount / 100).toLocaleString()}</b>
```

Comment:
Good call with `toLocaleString()`, we could even use [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) like we do [here](https://github.com/sendchinatownlove/sendchinatownlove.github.io/blob/aec06e27371864100f4bb0e5a93a77e9c9c3186c/src/pages/MerchantVoucherDashboard/MerchantVoucherDashboard.tsx#L49) if we want to be really fancy